beluga/models.py

```diff
@@ -8,7 +8,7 @@ class User(Base):
 
     __tablename__ = 'users'
 
-    id = sa.Column(sa.Integer, primary_key=True)
+    id = sa.Column(sa.BigInteger, primary_key=True)
```

Comment:
good call here--i was thinking there might've been some miscellaneous voodoo going on to convert from UUID to integer but i think this is the right approach (assuming we're just going to store the 128 bit number in here)

Reply:
Yep, just stealing the eventbrite ID. I think its best to keep both our own unique id and the eventbrite unique id, but this is laziness + MVP territory.